numpy/core/multiarray.pyi

```diff
@@ -1012,3 +1012,9 @@ class flagsobj:
     def owndata(self) -> bool: ...
     def __getitem__(self, key: _GetItemKeys) -> bool: ...
     def __setitem__(self, key: _SetItemKeys, value: bool) -> None: ...
+
+class _CopyMode(enum.Enum):
+
+    ALWAYS: L[1]
+    IF_NEEDED: L[0]
+    NEVER: L[2]
```

Comment:
This will need to be imported from the main numpy namespace instead,
as currently you have two distinct enums named `_CopyMode`.